src/lwc/gsChecklist/gsChecklist.html

```diff
@@ -0,0 +1,55 @@
+<template>
+    <article class="slds-setup-assistant__step">
+        <div class="slds-summary-detail">
+            <button class="slds-button slds-button_icon slds-m-right_x-small slds-m-top_x-small"
```

Comment:
can't we use a lighting-button here?

Reply:
I try use lighting-button but break all style in the button